src/transactions/FeeBumpTransactionFrame.cpp

```diff
@@ -551,4 +551,10 @@ FeeBumpTransactionFrame::toStellarMessage() const
     msg.transaction() = mEnvelope;
     return msg;
 }
+
+uint64_t
+FeeBumpTransactionFrame::getSize() const
+{
+    return static_cast<uint64_t>(xdr::xdr_size(mEnvelope));
```

Comment:
I'm unsure if this is the correct approach for `FeeBumpTransactionFrame`

Reply:
this is unused in FeeBumpTransactionFrame, right? I agree with Dima's suggestion that we should just extract size calculation as a private method of TransactionFrame.